tests/test_past_kv.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+import requests
+import torch
+from PIL import Image
+from transformers import AutoProcessor, LlavaForConditionalGeneration
+
+from peft import PrefixTuningConfig, get_peft_model
+
+
+class TestPastKV(unittest.TestCase):
+    def test_past_kv(self):
+        model_id = "trl-internal-testing/tiny-random-LlavaForConditionalGeneration"
```

Comment:
Let's add a comment to the start of the test to explain it. E.g. something like `# Ensure that models like Llava that pass past_key_values automatically do not fail, see #1938`.